sandbox/gvisor/run.py

```diff
@@ -162,7 +168,7 @@ def preserve(*locations, short_failure=False):
 
 preserve("/usr/local/lib")
 if os.path.exists('/lib64'):
```

Comment:
Would it be clearer to add the condition and explanation directly here: `if os.path.exists('/lib64') and not os.path.islink('/lib64')`?

Reply:
Good idea, made that change.